resources/features/Sampling.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Making a Sample
+    The sample creation method will ingest a frame and a set of stratification parameters to produce an output sample.
+    To do this, the pre-filtered dataset is sorted in ascending order by 'prn'. Each of the stratification stratas with
+    a selection type parameter set as 'Census' or 'Prn Sampling' (ommitting 'Universal') is then processed. We then use
+    the start point parameter (of a given strata), and select units in prn order until the required sample size is
+    reached. All samples are outputted collectively as a DataFrame, with an appended 'Cell Number' to each to indicate
+    the strata to which it belongs.
+
```

Comment:
The scenarios seem to imply that there will be separate data frames by selection type, however the requirement is for a collated output i.e. a single data frame containing all 'Census' and 'PRN' sampled units + exported as csv

Reply:
Maybe I should reword this. Currently for each strata a sample is created and appends this with the given CellNumber from the strata and then the next strata is appended to the existing strata DataFrame until all strata are either processed or some omitted.